frontend/src/pages/add/VendorAdd.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { FormEvent, useState } from "react";
+import React, { FormEvent, useEffect, useState } from "react";
```

Comment:
useEffect isn't implemented yet, but I am gonna let you leave it in

Reply:
This will be used soon